src/main/java/sirius/tagliatelle/emitter/BlockEmitter.java

```diff
@@ -43,19 +43,19 @@ public BlockEmitter(Position startOfBlock, String name, Emitter alternative) {
     @Override
     protected void emitToContext(LocalRenderContext context) throws Exception {
         if (context.blockExists(name)) {
-            emitDebugMessage(context, "start rendering block '" + name + "'");
```

Comment:
if (blockExists) {
 log (start rendering...)
context.emitBlock(name);
log(finished...);
return;
}

if (alternative != null) {
log(...)
alternative.emit(context);
log(...)
return;
}

emitDebugMessage(context, "requested block '%s' does not exist", name);

Reply:
Done.

Note: also adapted `LocalRenderContext.emitBlock` to use blockExists, avoiding duplicated logic if: 

- blocks == null
- blocks.get(name) == null -> _containsKey_